proto/http_route.proto

```diff
@@ -146,3 +146,14 @@ message Ratio {
   // MUST not be zero.
   uint32 denominator = 2;
 }
+
+// A range of HTTP response status codes, inclusive.
+message StatusRange {
+  uint32 min = 1;
+  uint32 max = 2;
+}
```

Comment:
I thought about this, but I'm worried about backwards compatibility.  Since the types are structurally identical, I think it should be fine, but it gave me the heebie-jeebies so I left it alone.

Reply:
Oh, on second reading I guess you're suggesting using the type defined in destionation.proto in the outbound policy.  I think as a matter of hygiene the plan is to eventually deprecate the getProfile API so depending on its types feels weird.

Reply:
it seems like ideally there could be one type in `http_types.proto`, but the ship has sailed on that. :woman_shrugging: